frame/staking/src/pallet/mod.rs

```diff
@@ -1616,6 +1616,27 @@ pub mod pallet {
 			Self::chill_stash(&stash);
 			Ok(())
 		}
+
+		/// Force a validator to have at least the minimum commission. This will not affect a
+		/// validator who already has a commission greater than or equal to the minimum.
+		#[pallet::weight(T::WeightInfo::force_apply_min_commission())]
+		pub fn force_apply_min_commission(
+			origin: OriginFor<T>,
+			validator_stash: T::AccountId,
+		) -> DispatchResult {
+			ensure_root(origin)?;
```

Comment:
should be `ensure_signed`?

Reply:
[5ec80f2](https://github.com/paritytech/substrate/pull/10786/commits/5ec80f20623f8c9507dfd455b0db77bda26e3109)